packages/medusa/src/loaders/helpers/resolve-plugins.ts

```diff
@@ -126,7 +133,8 @@ export function getResolvedPlugins(
     ]
   }
 
-  const resolved = plugins.map((plugin) => {
+  const extensionDirectoryPath = "dist"
+  const resolved = configModule?.plugins.map((plugin) => {
```

Comment:
sugg: This might break if plugins are not added to the config

```suggestion
  const resolved = (configModule?.plugins || []).map((plugin) => {
```

Reply:
Yeah. Makes sense

Reply:
Okay. Just checked. It looks like the `plugins` property in `ConfigModule` type is required. Also configLoader returns an empty plugins array when one is not defined.

https://github.com/medusajs/medusa/blob/develop/packages/core/types/src/common/config-module.ts#L785

https://github.com/medusajs/medusa/blob/develop/packages/medusa/src/loaders/config.ts#L104

But still, if I am missing something, then do let me know :)